source/modules/MEI.xml

```diff
@@ -227,6 +239,12 @@
         <valItem ident="fd">
           <desc xml:lang="en">Three quarter-tones flat.</desc>
         </valItem>
+        <valItem ident="xu">
+          <desc xml:lang="en">Five quarter-tones sharp.</desc>
+        </valItem>
+        <valItem ident="ffd">
+          <desc xml:lang="en">Five quarter-tones flat.</desc>
+        </valItem>
```

Comment:
Would we need `xd` and `ffu` as alternatives for three quarter-tone sharp/flat as well?

Reply:
We already have three-quarter values in the gestural range.